Python/pywarpx/picmi.py

```diff
@@ -777,7 +778,7 @@ def initialize_inputs(self):
         self.diagnostic.format = self.format
         self.diagnostic.openpmd_backend = self.openpmd_backend
         self.diagnostic.dump_rz_modes = self.dump_rz_modes
-        self.diagnostic.period = self.period
+        self.diagnostic.intervals = self.intervals
```

Comment:
This should continue to use the PICMI standard for input, so this line should be
`self.diagnostics.intervals = self.period`